tests/cloud_test.py

```
@@ -731,6 +733,102 @@ async def test_mideaaircloud_login_invalid_user(self) -> None:
+        # that decrypts to a small valid lua snippet.
+        res.text = AsyncMock(
+            return_value=(
+                "f424cd84479c665a7e8a82d3b6bea6b67a1fdc95a7783791a6ff35b2953a158a"
+            ),
+        )
+        session.get = AsyncMock(return_value=res)
+        cloud = get_midea_cloud(
+            "Midea Air",
+            session=session,
+            account="account",
+            password="password",
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "password"

(S106)